### ISSUE TITLE
Support X Axis Time Scaling

### ISSUE THREAD

**Issue by beeme1mr** [artifacthealth/phantom-chartjs#1]
The x axis time scale option requires moment.js.  It would be very handy this these formatting options readily available.

http://www.chartjs.org/docs/#scales-time-scale

phantom-chartsjs throws the following exception:

```
InternalServerError: Error: Chart.js - Moment.js could not be found! You must include it before Chart.js to use the time scale. Download at https://momentjs.com
    phantomjs://code/Chart.min.js: 14 (in function 'initialize')
    phantomjs://code/Chart.min.js: 12 (in function 'Element')
    ...
```

**Comment by meirgottlieb** [comment]
Can you please provide an example chart configuration for my testing that uses this option?

**Comment by beeme1mr** [comment]
Here you go.

https://gist.github.com/beeme1mr/88348f4fb978f7e51af1ad377ca3b8e1

**Comment by beeme1mr** [comment]
I think the time I provided in that last example won't work.  It most likely needs to be epoch time.  I'll run a few tests on my end.

**Comment by beeme1mr** [comment]
@meirgottlieb Would it be possible for this to be published to NPM?

**Comment by meirgottlieb** [comment]
Just published as 0.2.3.
